packages/components/src/mobile/keyboard-aware-flat-list/shared.native.js

```diff
@@ -0,0 +1,6 @@
+export const OPTIMIZATION_PROPS = {
+	windowSize: 17,
+	maxToRenderPerBatch: 15,
+};
+
+export const OPTIMIZATION_ITEMS_THRESHOLD = 30;
```

Comment:
Potentially, we may want to leave a note or comment with some further context on these values (as they deviate from the [default values](https://reactnative.dev/docs/optimizing-flatlist-configuration)). Not a blocker for this PR, however.

Reply:
Good idea! Updated in https://github.com/WordPress/gutenberg/pull/59833/commits/a9dc932ca52d22507e3b7b08a7cc080408d24005